<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>account_settings</class>
 <widget class="QWidget" name="account_settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(34, 40, 49);</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>600</width>
     <height>290</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(57, 62, 70);
border-radius: 10px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_confirm_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>Old Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>Confirm Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_new_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_old_password">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>360</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(34, 40, 49);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>260</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(238, 238, 238);
padding-top: 3px;
padding-left: 3px;</string>
    </property>
    <property name="text">
     <string>CHANGE MASTER PASSWORD</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="cancel_button">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(34, 40, 49);
}
QPushButton:hover:!pressed {
   background-color: rgb(150, 150, 150);
}</string>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>220</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
    background-color: rgb(136, 8, 8);
}
QPushButton:hover:!pressed {
   background-color: rgb(255, 0, 0);
}</string>
    </property>
    <property name="text">
     <string>Delete Account</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
